components/massmint/OnBoarding.vue

```diff
@@ -48,7 +74,7 @@
 
     <div class="is-flex is-justify-content-center my-8">
       <span
-        v-for="(card, index) in cards"
+        v-for="(_, index) in cards"
```

Comment:
Avoid declaring unused variables and prefer self closing tags

```
<span
        v-for="index in cards.length"
        :key="index"
        class="carousel-dot mx-2"
        :class="{ 'is-active': index - 1 === currentSlide }" />
```

Reply:
Notice the index - 1? That's because when iterating over a number, Vue starts the index at 1, whereas when iterating over an array, it starts at 0. So we adjust for that to keep the logic the same.